code/modules/admin/sql_notes.dm

```diff
@@ -133,15 +160,18 @@
 			output += " <a href='?_src_=holder;shownoteckeylinkless=[target_ckey]'>\[Refresh Page\]</a></center>"
 		output += ruler
 		while(query_get_notes.NextRow())
-			var/id = query_get_notes.item[1]
+			var/secret = text2num(query_get_notes.item[1])
+			if(linkless && secret)
+				continue
```

Comment:
ya, the mere presence of secret notes would just piss players off and call all sorts of drama as they guess and complain about it. If they notice a hidden note like a day or so after a forum argument with an admin, they will start accusing the admin of leaving it in retaliation for the forum argument.